### ISSUE TITLE
Possible Draconic Evolution Bug

### ISSUE THREAD

**Issue by Watschman** [imrgr/rise-of-the-pack#14]
Pack Version: Former 2.0
Java Version: 8-153 (64bit)
Additional Mods Added: Fullscreen Windowed (Borderless) for Minecraft
Type of issue: Possible misconfiguration 
-------------------------
Hello, 
haven't heard of you for a long time, I am getting close to the end and encountered a problem/possible misconfiguration with Draconic Evolution's Draconic Armor.
I am unable to fly with just the full Draconic Armor, therefore I looked through the configs and noticed that flight is disabled. I thought "well okay, maybe you want the people to use the Angel Ring" but then I encountered a nice dragon called the "Chaos Guardian" who was a real pain in the butt to kill and ended up in 10 attempts within multiple hours due to the fact that the Draconic Bow is useless and you are unable to effectively catch him with the speed of the angel Ring. Thanks to my 10 max upgraded Draconic Capacitors I was able to survive the glorious battle and slew my first Chaos Guardian. Now I can't imagine you want to torture people by doing this over and over again due to the fact that you need to slay at least 15 of these guardians as far as I have seen to complete the quests, so I took the might of god in my hand and enabled the flight ability myself. It is a lot more reasonable with that sorta speed.
Now the question, is it intended to be that tedious or just a misconfiguration on you part?
So long, 
Watschman.

**Comment by tloskpd** [comment]
Yeah I encountered that issue as well, but I have not gone against the Chaos Dragon yet, still upgrading armor.  Even in the quest book where it wants you to kill the chaos dragon, it tells you to increase the flight speed in the armor gui which does not matter since, as you said, flight is disabled in the config.  Like you, I modified my server to allow draconic flight.

**Comment by Watschman** [comment]
Well I finished the Questbook a couple days ago and therefore the pack and move on. Overall it was an enjoyable experience, even though I got frustrated at times but that was mostly due to bugs or poorly optimized features in the Mods themselves and not fault of the modpack. I learned quite a few things, not only about mods in 1.12 but also about Vanilla. I had not clue that Invokation Villagers and End Cities were a thing.
Dunno why the Modpack Author didn't respond to anything for almost over month now but well,  hopefully some bugs that I encountered during my playthrough will be fixed within the next 4-5 months when I am starting the next Minecraft Project.
